### PR TITLE
Add case and incident tactics

### PR DESCRIPTION
**Pull request by dosiso**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Review by ityankel — APPROVED** [review]

File: specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json [hunk 1]
```diff
@@ -4419,6 +4419,14 @@
           "readOnly": true,
           "type": "array"
         },
+        "tactics": {
```
Comment (lirenhe) [inline specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json]: Not sure whether this is a good practice to have 2 'tactics' with the same definition.

Comment (dosiso) [reply specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json]: I'm don't completely understand the 2 'tactics' with the same definition.
There's one definition for CaseProperties which is a property of Case that is to be deprecated in a few months, and a definition for IncidentAdditionalData which is a property of Incident which is the one to replace Case.
If it still doesn't make sense can you elaborate?

Comment (lirenhe) [reply specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json]: I see. thanks

**Review by ravbhatnagar — COMMENTED** [review]
tactics readonly prperty is being added to cases and incidents. Signing off from ARM side.

**Comment by dosiso** [comment]
I'm still unable to merge, please advise @ravbhatnagar

**Review by lirenhe — APPROVED** [review]

**Comment by lirenhe** [comment]
@dosiso , for the pending job, have you signed the CLA agreement?  https://cla.opensource.microsoft.com/

**Comment by dosiso** [comment]
@lirenhe , should I sign the CLA as a Microsoft employee (Azure Sentinel)? If so, by the way, we should probably check the auto-tests as I believe all the checks have completed before.

**Comment by lirenhe** [comment]
> @lirenhe , should I sign the CLA as a Microsoft employee (Azure Sentinel)? If so, by the way, we should probably check the auto-tests as I believe all the checks have completed before.

So do you mean you already signed it but still got the license/cla pending?

**Comment by dosiso** [comment]
I mean it should be automated so I don't have to do it.. @lirenhe

**Pull request merged**